kore/src/main/scala/org/kframework/rewriter/Rewriter.scala

```diff
@@ -15,31 +16,49 @@ trait Rewriter {
   //  def step(k: K): K
 
   /**
-   * (disregard this javadoc comment for now)
-   * Takes one rewriting step.
-   * - for regular execution, it returns the next K or False (i.e. an empty Or)
-   * - for symbolic execution, it can return any formula with symbolic constraints
-   * - for search, it returns an Or with multiple ground terms as children
+   * (disregard this javadoc comment for now) Takes one rewriting step.
+   *   - for regular execution, it returns the next K or False (i.e. an empty Or)
+   *   - for symbolic execution, it can return any formula with symbolic constraints
+   *   - for search, it returns an Or with multiple ground terms as children
```

Comment:
Perhaps we should look for all the comments to see where we can add a new line to keep the semantic of separate ideas.

Reply:
Adding `<p>` tags should fix this, I'll make the changes